src/ProjectOrigin.Wallet.Server/Models/Wallet.cs

```diff
@@ -0,0 +1,6 @@
+using System;
+using ProjectOrigin.Wallet.Server.HDWallet;
+
+namespace ProjectOrigin.Wallet.Server.Models;
+
+public record WalletA(Guid Id, string Owner, IHDPrivateKey PrivateKey);
```

Comment:
We have chosen to use Records since it is the KISS solution when also using dapper, since it has no way to track changes, so we would not update the entity anyway.